### PR TITLE
Add suitable description for `authorized-by`

### PR DESCRIPTION
**Pull request by yogeshbdeshpande**
Fixes issue #72

### CODE REVIEW

File: draft-ietf-rats-corim.md [hunk 1]
```diff
@@ -850,6 +850,9 @@ identified by a class identifier have measurements that are common to the
 class. Environments identified by an instance identifier have measurements that
 are specific to that instance.
 
+Supply chain entity responsible for the measurements can optionally assert their cryptographic
```
Comment (nedmsmith) [inline draft-ietf-rats-corim.md:853]:
```suggestion
The supply chain entity that is responsible for providing reference or endorsed measurement values is, by default, the CoRIM signer. If a different entity is authorized to provide measurement values, the `authorized-by` statement can be supplied.
```

File: draft-ietf-rats-corim.md [hunk 2]
```diff
@@ -850,6 +850,9 @@ identified by a class identifier have measurements that are common to the
...
+identity via the use of `authorized-by` field of the `measurement-map`
```
Comment (nedmsmith) [inline draft-ietf-rats-corim.md:854]:
```suggestion
The `authorized-by` statement is contained in `measurement-map`.
```

File: draft-ietf-rats-corim.md [hunk 3]
```diff
@@ -862,6 +865,9 @@ The following describes each member of the `measurement-map`:
 * `mval` (index 1): The measurements associated with the (sub-)environment.
   Described in {{sec-comid-mval}}.
 
+* `authorized-by` (index 2): The cryptographic identity of the individual or organization that is
+ the designated authority for this measurment. For example, producer of the measurement or a delegated entity.
```
Comment (nedmsmith) [inline draft-ietf-rats-corim.md:869]:
```suggestion
 the designated authority for this measurement. For example, producer of the measurement or a delegated supplier.
```

**Review by nedmsmith — COMMENTED** [review]
See inline suggested changes.

**Review by nedmsmith — APPROVED** [review]

File: draft-ietf-rats-corim.md [hunk 4]
```diff
@@ -850,6 +850,11 @@ identified by a class identifier have measurements that are common to the
 class. Environments identified by an instance identifier have measurements that
 are specific to that instance.
 
+The supply chain entity that is responsible for providing the the measurements (i.e. Reference Values or Endorsed Values)
```
Comment (thomas-fossati) [inline draft-ietf-rats-corim.md:853]:
```suggestion
The supply chain entity that is responsible for providing the measurements (i.e. Reference Values or Endorsed Values)
```

**Review by henkbirkholz — APPROVED** [review]

**Review by thomas-fossati — APPROVED** [review]
LGTM, modulo a small typo (i.e., `the the`)

**Pull request merged**